nikola/plugins/task/galleries.py

```diff
@@ -94,8 +94,8 @@ def set_site(self, site):
         for source, dest in self.kw['gallery_folders'].items():
             if source in appearing_paths or dest in appearing_paths:
                 problem = source if source in appearing_paths else dest
-                utils.LOGGER.error("The gallery input or output folder '{0}' appears in more than one entry in GALLERY_FOLDERS, exiting.".format(problem))
-                sys.exit(1)
+                utils.LOGGER.error("The gallery input or output folder '{0}' appears in more than one entry in GALLERY_FOLDERS, ignoring.".format(problem))
+                continue
```

Comment:
Why ignoring this instead of raising an exception? This will with high probability (haven't checked to be 100% sure) lead to an error as the plugin tries to create `index.html` in the output folder twice.